File: official/docs/java/v6/carrier-accounts/delete.java
package carrier_accounts;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        client.carrierAccount.delete("ca_...");
    }
}

File: official/docs/node/current/carrier-accounts/delete.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  const carrierAccount = await client.CarrierAccount.retrieve('ca_...');

  await client.CarrierAccount.delete(carrierAccount.id);
})();
